clusterloader2/testing/load/modules/reconcile-objects.yaml

```diff
@@ -0,0 +1,268 @@
+## Input params
+# Valid actions: "create", "scale and update", "delete"
+{{$actionName := printf "%s objects" .actionName}}
+{{$namespaces := .namespaces}}
+{{$tuningSet := .tuningSet}}
+
+# Derivative variables
+{{$is_creating := (eq .actionName "create")}}
```

Comment:
Nit: you don't need is_creating and is_scaling anymore :)

Reply:
Thanks, removed. :)